docs/DeveloperGuide.adoc

```diff
@@ -789,6 +789,24 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 * prefers typing over mouse input
 * is reasonably comfortable using CLI apps
 
+**Feature contributions**:
+
+Umar:
+
+* A major feature that I am doing is
```

Comment:
there is a lot more to the deduplication feature that you mentioned, because what you're talking about can probably be done within 1 hour of work. But okay for now